spec/features/admin/configuration/tax_rates_spec.rb

```diff
@@ -1,12 +1,12 @@
-# frozen_string_literal: true
-
 require 'spec_helper'
 
 describe "Tax Rates" do
   include AuthenticationHelper
 
   let!(:calculator) { create(:calculator_per_item, calculable: create(:order)) }
-  let!(:tax_rate) { create(:tax_rate, calculator: calculator) }
+  let!(:tax_rate) { create(:tax_rate, name: "IVA", calculator: calculator) }
+  let!(:zone_id) { create(:zone, name: "Ilhas") }
+  let!(:tax_category_id) { create(:tax_category, name: "Full") }
```

Comment:
I think these should be named `:zone` and `:tax_category`, as they are full objects and not just ids.